src/content/pages/volunteers.mdx

```diff
@@ -3,26 +3,32 @@ title: Volunteers
 subtitle: We need your help to make EuroPython 2024 a success!
 ---
 
-# Volunteering@EuroPython
+# Volunteering @ EuroPython
 
 Python owes much of its success to open source and the community surrounding it. In the same spirit, EuroPython is organized and run by volunteers from the Python community. Alas, we're only a few and need your help to make the show run as smoothly as possible. Each year we are joined by our awesome attendees as volunteers to help in running this wonderful conference. This page explains some of the ways you can help us.
 
 Being a volunteer means giving your time for free, without expecting anything back, just for the pleasure of being a part of this amazing conference. It's a great way to meet new people, learn new things, and have a lot of fun. You can volunteer for a few hours or for the whole conference, it's up to you.
 
 We will also have a small number of perks for you, some small gifts to say "thank you" for your time and for your help.
 
+
 # Things to remember
 
-* This is in person volunteering and would require you to be present at the venue.
+* This is **in-person** volunteering and would require you to be present at the venue.
 * While we strive to make the conference accessible to everyone, we cannot provide travel or accommodation support to volunteers.
 * To enter the conference you need a ticket.
-* You are welcome to volunteer as much or as little as you want to. We appreciate any help you can give us.
+* You are welcome to volunteer as much or as little as you want to. We appreciate any help you can give us. However please note we will not be able to refund you for the ticket, should you choose to volunteer for ten (10) hours or more. We are unable to commit to that at this point.
```

Comment:
This is not correct. We can refund tickets if people choose to switch over to a volunteer ticket.

Reply:
Got this one from the last year's website. It says:


> You are welcome to volunteer as much or as little as you want to, even if you already hold a ticket. However please note we will not be able to refund you for the ticket should you choose to volunteer for eight hours or more. We are unable to commit to that at this point.

Reply:
We can **not** do it **after** the event. 
But we can do it before. Or better they can cancel their ticket like everybody else before the event (preferably after they got a voucher).

Reply:
Thanks ❤️